sw/device/lib/testing/pinmux_testutils.c

```diff
@@ -42,6 +42,21 @@ void pinmux_testutils_init(dif_pinmux_t *pinmux) {
   CHECK_DIF_OK(dif_pinmux_output_select(pinmux, kTopEarlgreyPinmuxMioOutIoc4,
                                         kTopEarlgreyPinmuxOutselUart0Tx));
 
+  // Enable pull-ups on UART0 TX/RX
+  dif_pinmux_pad_attr_t out_attr;
```

Comment:
i need to double check the DIFs, but its feels like we should be able to pass NULL pointers for outputs hehe..if we don't care about it.

Reply:
Either way is fine, but the DIFs generally want the callers to do something with all outputs. In this case, the output explains which attributes were actually accepted, so it's still kind of at the validation level.

However, it's arguable that this validation mechanism really doesn't need to be done at run time, that it ought to be a data sheet exercise only.